components/Sequence/basic_usage.rst

```diff
@@ -0,0 +1,70 @@
+Basic Usage
+===========
+
+SequenceSubjectInterface
+------------------------
+
+Let's see how an exemplary class implementing **SequenceSubjectInterface** should look like.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Sequence\Model\SequenceSubjectInterface;
+
+    class Order implements SequenceSubjectInterface
+    {
+        /**
+         * @var string
+         */
+        private $number;
+
+        /**
+         * {@inheritdoc}
+         */
+        public function getSequenceType()
+        {
+            return 'order';
+        }
+
+        /**
+         * {@inheritdoc}
+         */
+        public function getNumber()
+        {
+            return $this->number;
+        }
+
+        /**
+         * {@inheritdoc}
+         */
+        public function setNumber($number)
+        {
+            $this->number = $number;
+        }
+    }
+
+SequentialGenerator
+-------------------
+
+The **SequentialGenerator** is used to provide a sequence for a given subject, that implements the **SequenceSubjectInterface**.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Sequence\Model\Sequence;
+    use Sylius\Component\Sequence\Number\SequentialGenerator;
+    use Order;
```

Comment:
I think this will be for second iteration on docs, but we should agree on a single sample namespace for all custom code.

I'd strongly suggest AppBundle, just like Symfony does. This is recommended project structure anyway.
So that would become ``AppBundle\Entity\Order``.